lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +86,20 @@ def compare_profiles(
     :param profile_to_compare: a dictionary of a profile to compare the unknown profile to
     :return: the distance between the profiles
     """
+    if not (
+            isinstance(unknown_profile, dict) and isinstance(profile_to_compare, dict) and 'name' in unknown_profile
+            and 'freq' in unknown_profile and 'name' in profile_to_compare and 'freq' in profile_to_compare
+    ):
+        return None
+
+    else:
+        unknown_profile_freq = unknown_profile['freq']
+        profile_to_compare_freq = profile_to_compare['freq']
+        actual = list(unknown_profile_freq.values())
+        predicted = list(profile_to_compare_freq.values())
+        mse = calculate_mse(predicted, actual)
+        rounded_mse = round(mse, 3)
```

Comment:
Is this the right place for rounding?

Reply:
Is rounding even necessary? i find out that code works without it at all.

Reply:
It is expected at the very last step of the laboratory, for mark 10